lib/fluent/plugin/out_influxdb2.rb

```diff
@@ -141,12 +141,24 @@ def write(chunk)
       point.time(time_formatted, @precision)
       points << point
     end
-    @write_api.write(data: points)
+    @write_api.write(data: points, bucket: bucket)
     log.debug "Written points: #{points}"
   end
 
   private
 
+  def expand_placeholders(chunk)
+    bucket = extract_placeholders(@bucket, chunk)
+
+    if @measurement
+      measurement = extract_placeholders(@measurement, chunk)
+    else
+      measurement = chunk.metadata.tag
+    end
+
+    return bucket, measurement
+  end
+
```

Comment:
Please use correct code style:

```suggestion
  def expand_placeholders(chunk)
    bucket = extract_placeholders(@bucket, chunk)

    measurement = if @measurement
                    extract_placeholders(@measurement, chunk)
                  else
                    chunk.metadata.tag
                  end

    [bucket, measurement]
  end
```

You can test code formatting by: `bundle exec rake rubocop`

Reply:
Thanks for the suggestion, I've update the code style.